cocinaborcelle/src/pages/api/excluirConta.ts

```diff
@@ -0,0 +1,34 @@
+import { auth } from "@/../auth/lucia";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+	if (req.method != "POST") {
```

Comment:
```suggestion
	if (req.method != "DELETE") {
```

Já que está excluindo uma instância do banco de dados, utilize o método http delete. Utilizar os métodos certos ajuda em depurações futuras, para quando for olhar o que deu errado no console ou em network do browser